tests/test_login.py

```diff
@@ -3,24 +3,87 @@
 import requests
 import time
 
-from tests import conftest
+from pages.moderator import Moderator
+from pages.mozillians import Mozillians
+from pages.reps import Reps
+from pages.standups import Standups
 from pages.rptest import RpTest
+from tests import conftest
 
 
 class TestLogin:
 
     @pytest.mark.nondestructive
-    def test_login(self, selenium, ldap, token, urls):
+    def test_login_testrp(self, selenium, ldap, token, urls):
         test = RpTest(selenium, urls['testRP'])
         test.login_with_ldap(ldap['email'], ldap['password'], conftest.passcode(ldap['secret_seed']))
         access_token = conftest.access_token(token)
         api = token['api']
         user_logs = requests.get(api, headers={'Authorization': 'Bearer {0}'.format(access_token)})
         initial_user_log_date = json.loads(user_logs.content)[0]['date']
-        time.sleep(16)
+        time.sleep(900)
```

Comment:
Either use a config entry for that or add a variable in the top of the file like:

```
SLEEP_TEST_LOGIN = 900
```